React icons documentation update (#20862)
microsoft/fluentui: apps/public-docsite/src/pages/Styles/FabricIconsPage/docs/web/FabricIconsUsage.md

You can use Fluent UI's icons in a few ways, depending on if you're using React Icons, Fluent UI React or Fabric Core.

> ### React Icons: SVG Based React Icons

Reviewer: nit, Microsoft style is to use sentence case (though we're very inconsistent about it today)
```suggestion
### React Icons: SVG-based React icons
```

The Fluent UI SVG icon set is released under the MIT License.

> `@fluentui/react-icons` are SVG based icons wrapped in a React element. Because each icon is its own element, you only need to import what you need for your application.

Reviewer: ```suggestion
The `@fluentui/react-icons` package contains SVG-based icons wrapped in a React component. Because each icon is its own component, you only need to import and bundle the icons used in your application.
```

There are two different states of each icon, `Filled` and `Regular`, so you can choose what works best for your application. These are by default sized to `1em`, and can be scaled up or down to suit your developer needs.

> In order to use these icons, simply import them as `import { [Componentname][state] } from @fluentui/react-icons` as follows:

Reviewer: ```suggestion
To use these icons, simply import them as `import { [IconName][Style] } from @fluentui/react-icons`. For example:
```

```tsx
import { AccessTimeFilled } from '@fluentui/react-icons';
```

> The previous icon offerings are scalable, but if you know what size you want to use for your icons, and you are not planning on using multiple sizes, it is recommended to use the sized icons. These are the same icons as the general case, but in multiple different sizes, built to look pixel perfect at those specific sizes.
> You can import the sized icons in a similar way:

Reviewer: ```suggestion
If you know what size you want to use for your icons, and they don't need to be resized dynamically, it is recommended to use the sized icons. These are the same icons as the general case, but in multiple different sizes, built to look pixel-perfect at each size.

You can import the sized icons in a similar way:
```

```tsx
import { AccessTime24Filled } from '@fluentui/react-icons';
```

> You can also style the icons using the `FluentIconsProps` interface, with the `className` prop or the `primaryFill` prop.

Reviewer: ```suggestion
Each icon also accepts `className` and `primaryFill` props for styling.
```
